src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(y = 1) {
+      if (y < 0) {
+        return this;
+      }
+
+      this.coords.y += y;
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(y = 1) {
      if (y > 0) {
        this.coords.y += y;
      }

      return this;
    },
```
Just FYI - it's shorter to write it in this way